source/assets/scripts/recipe-detail.js

```diff
@@ -78,3 +79,18 @@ function bindFoodieButton() {
     window.location.assign('foodie.html');
   });
 }
+
+function goBack(){
+  const btn = document.getElementById('white-arrow');
+  let index = document.referrer.lastIndexOf('/');
+  let str = document.referrer.substring(index + 1);
+  console.log(str);
+
+  btn.addEventListener('click', () => {
+    if (str === 'index.html'){
+      window.location.assign('index.html');
+    } else {
+      window.history.back();
```

Comment:
I am not sure that is the behavior we want. For example, if we go in to foodie mode and then click exit. Clicking this back button will bring us back in the foodie mode even if the user might want go back to the homepage.